wagtailvideos/apps.py

```diff
@@ -1,23 +1,22 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from django.apps import AppConfig
-from django.core.checks import Error, register
+from django.core.checks import Warning, register
 
-from wagtailvideos.utils import which
+from wagtailvideos.utils import ffmpeg_installed
 
 
 def ffmpeg_check(app_configs, path=None, **kwargs):
-    errors = []
-    if which('ffmpeg', path=path) is None:
-        errors.append(
-            Error(
-                'ffmpeg could not be found on your system, try installing it.',
+    messages = []
+    if not ffmpeg_installed(path=path):
```

Comment:
`ffmpeg_check()` will never be called with a `path` argument by Django (at least according to the Django docs which make no mention of any arguments), so the default `None` value flows through the whole call chain to `which`. Can this parameter just be removed?